docs/UserGuide.md

```diff
@@ -1401,10 +1401,213 @@ Command Options: This command does not take in any arguments and will not proces
 >
 > <img src="images/unicash/command-outputs/get-budget/getBudgetNoBudgetSuccess.png" width="1000" />
 
+[//]: # (TODO: maybe add failed case if more arguments provided)
+
 ### 4.3 Financial Statistics Features
 
 #### 4.3.1 GetTotalExpenditureCommand
 
+Retrieves the total expenditure by month with optional filters for category and year. Also filters the transactions
+by the given month, year, and category.
+
+Use `list` to view all transactions again.
+
+Command: `get_total_expenditure month/Month [c/Category] [year/Year]`
+
+Command Options:
+
+| Option Name | Optional? | Purpose                                                                               |
+|-------------|-----------|---------------------------------------------------------------------------------------|
+| month/      | No        | Month to calculate the total expenditure.                                             |
+| c/          | Yes       | Category of expenditure to retrieve.<br>Defaults to all categories if not provided.   |
+| year/       | Yes       | Year to calculate the total expenditure.<br>Defaults to current year if not provided. |
```

Comment:
Want to mention it cateogyr is case insensitive?

Reply:
Sure